frontend/public/components/deployment.jsx

```diff
@@ -117,7 +129,61 @@ const DeploymentsDetailsPage = props => <DetailsPage
 />;
 
 const Row = props => <WorkloadListRow {...props} kind="Deployment" actions={menuActions} />;
-const DeploymentsList = props => <List {...props} Header={WorkloadListHeader} Row={Row} />;
+
+const kind = 'Deployment';
+
+const DeploymentTableRow = (o, index) => {
+  return {
+    id: o.metadata.uid,
+    cells: [
+    {
+      title: <ResourceLink kind={kind} name={o.metadata.name} namespace={o.metadata.namespace} title={o.metadata.uid} />,
+    },
+    {
+      title: <ResourceLink kind="Namespace" name={o.metadata.namespace} title={o.metadata.namespace} />,
+    },
+    {
+      title: <LabelList kind={kind} labels={o.metadata.labels} />,
+    },
+    {
+      title: <Link to={`${resourcePath(kind, o.metadata.name, o.metadata.namespace)}/pods`} title="pods">
+        {o.status.replicas || 0} of {o.spec.replicas} pods
+      </Link>,
+    },
+    {
+      title: <Selector selector={o.spec.selector} namespace={o.metadata.namespace} />,
+    },
+    {
+      title: <ResourceKebab actions={menuActions} kind={kind} resource={o} />,
```

Comment:
It would be useful to extract `ActionsMenu` from PageHeading for easier use and sync ActionsMenu dropdown with this Kebap dropdown.

Reply:
This is a larger scope as discussed than just tables, so it is probably better to leave it for followups

Reply:
I think it probably best to treat as a stretch for this PR, and open a separate issue/pr to consolidate menu actions globally.